server/service/IndividualServicesService.js

```diff
@@ -357,27 +358,33 @@ exports.redirectInfoAboutApprovalStatusChanges = function (body, user, originato
       let applicationAddress = body["subscriber-address"];
       let applicationPort = body["subscriber-port"];
       let subscriberOperation = body["subscriber-operation"];
+      let applicationProtocol = body["subscriber-protocol"];
 
       /****************************************************************************************
        * Prepare logicalTerminatinPointConfigurationInput object to 
        * configure logical-termination-point
        ****************************************************************************************/
 
-      let operationList = [
-        subscriberOperation
-      ];
+      let operationNamesByAttributes = new Map();
+      operationNamesByAttributes.set("subscriber-operation", subscriberOperation);
+
+      let tcpObjectList = [];
+      let tcpObject = formulateTcpObjectForApplication(applicationProtocol, applicationAddress, applicationPort);
```

Comment:
can we declare the tcpObjectList directly instead of declaring the method.

Reply:
Done, Declared the tcpObjectList inside the method only & removed function formulateTcpObjectForApplication